src/Native/ServerSideHelpers.js

```diff
@@ -96,9 +96,18 @@ function triggerEvent(eventName, value, node){
     return A2(_elm_lang$core$Native_Json.run, events[eventName].decoder, value)
 }
 
+function stringify(node) {
+    if (node.type == "keyed-node") {
+        node.children = node.children.map(function(childNode){
+            return childNode["_1"]
+        });
+    }
```

Comment:
Just more convenient for looking at the structure there than in a decoder. Was going to change it after getting the tests...